src/components/Ressource/Montants.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div class="form__group" v-bind:key="type.meta.id">
-    <legend
-      ><h3 v-if="!withoutHeader">{{ type.meta.label }}</h3></legend
-    >
-    <YesNoQuestion class="form__group" v-model="singleValue">
+    <legend>
+      <h2 v-if="!withoutHeader">{{ type.meta.label }}</h2>
+    </legend>
```

Comment:
`legend` n'est adapté qu'aux `fieldsets`.
```suggestion
    <h2 v-if="!withoutHeader">{{ type.meta.label }}</h2>
```